src/sentry/mediators/sentry_app_installations/destroyer.py

```diff
@@ -34,11 +35,16 @@ def _destroy_service_hooks(self):
 
     def _destroy_installation(self):
         if self.notify:
-            InstallationNotifier.run(
-                install=self.install,
-                user=self.user,
-                action='deleted',
-            )
+            try:
+                InstallationNotifier.run(
+                    install=self.install,
+                    user=self.user,
+                    action='deleted',
+                )
+            except ReadTimeout:
+                # if we get a read timeout, just move on and delete the installation
```

Comment:
Do you want logging or metrics on this?

Reply:
@markstory We probably should have logs for any error. Is that not automatically captured at the request level?

Reply:
Not if you catch it and `pass`. The SDK never gets a chance to handle the error in this case. We can add:

```python
from sentry_sdk import capture_exception

...

except ReadTimeout as e:
    capture_exception(e)

...
```

though and it should send it along to Sentry.

Reply:
We capture 500s in the internal sentry project. However, if an integration partner is having downtime/timeout conditions do we need to be alerted?

Reply:
I'd say yes, if we have some way of doing something in that scenario, but as we are today, I don't think we do. I think it's worth having the data, though.